Backend/admin/src/App.js

```diff
@@ -31,17 +30,16 @@ function App() {
         </Header>
         <Content>
           <Page>
-            <Route exact path="/" component={() => <div>HOME PAGE</div>} />
-            <Route exact path="/login" component={Login} />
-            <Route exact path="/questionary" component={Questionary} />
             <Switch>
-              <Route exact path="/questions" component={Questions} />
-              <Route exact path="/questions/create" component={AddQuestion} />
-              <Route
-                exact
-                path="/questions/:id/edit"
-                component={EditQuestion}
-              />
+              <Route exact path="/" component={() => <div>HOME PAGE</div>} />
+              <Route path="/login" component={Login} />
```

Comment:
/login shouldn't be rendered here if the user is already auth .

Reply:
we can filter them based on userAuth data